articles/ip-address-configuration.md

```diff
@@ -31,6 +31,9 @@ The [IP addresses](/connectors/common/outbound-ip-addresses#power-platform) from
 
 You should also consult the [Limits and Configuration](limits-and-config.md) article for a supplemental listing for known IP addresses that Power Automate uses.
 
+>[!NOTE]
+> Starting June 2022, Logic Apps will enable availability zones in phases based on region. New IP addresses for this support are already available for the Logic Apps service and managed connectors. To make sure flows still work after this date, update your firewall configurations to include both the [IP addresses for Logic Apps](/azure/logic-apps/logic-apps-limits-and-config#firewall-ip-configuration) and the [IP addresses for managed connectors](/connectors/common/outbound-ip-addresses) in the corresponding region. For more information about availability zones, see [Azure Logic Apps - Availability Zone support is coming soon](https://techcommunity.microsoft.com/t5/integrations-on-azure-blog/azure-logic-apps-availability-zone-support-is-coming-soon/ba-p/3254456). 
```

Comment:
@llistephanie and @MSFTMan, I updated the date to May because Azure Logic Apps has already release preview for several Azure regions. I've also added the link for the published doc.

Reply:
@ecfan Ok. I'll merge now. Thanks!

Reply:
@MSFTMan, I don't see my updated suggestion in the PR yet. Any issues?